examples/deform/linearized_fixed_displacement.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Example using the operator of fixed-displacement linearized deformation."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+import numpy as np
+import odl
+
+
+# --- Create template and displacement field --- #
+
+
+# Discrete reconstruction space: discretized functions on the rectangle
+# [-1, 1]^2 with 100 samples per dimension.
+discr_space = odl.uniform_discr([-1, -1], [1, 1], (100, 100), interp='linear')
+
+# Create a product space for displacement field
+disp_field_space = discr_space.vector_field_space
+
+# Create a template, and set the template as an offset rectangle
+template = odl.phantom.cuboid(discr_space, [-0.5, -0.5], [-0.25, 0])
+
+# Define a displacement field that rotates an image clockwise by theta. Then it
+# shifts the image by hx to the right and by hy upwards.
```

Comment:
This example is good for educational purposes and to show that we can do rigid body motions, but for users expecting some action, it's rather boring. Why not use some simple Gaussian that tapers off fast enough?